pywbem/mof_compiler.py

```diff
@@ -1290,11 +1428,11 @@ def p_referenceInitializer(p):
         try:
             p[0] = p.parser.aliases[p[1]]
         except KeyError:
-            ce = CIMError(
-                CIM_ERR_FAILED,
-                _format("Unknown alias: {0!A}", p[1]))
-            ce.file_line = (p.parser.file, p.lexer.lineno)
-            raise ce
+            raise MOFParseError(
+                msg=_format(
+                    "Cannot compile reference initialization because "
+                    "it specifies an unknown alias: {0!A}", p[1]),
```

Comment:
How about change to ' instance alias not previously defined in the MOF file'.  That covers our issues about definition before use and also insures the user can find reference to instance alias vs. class alias in doc.

Reply:
DONE - changed to:
"Cannot compile reference initialization because the instance alias {0!A} was not previously defined in in the MOF"